source/guide_wekan.rst

```diff
@@ -0,0 +1,196 @@
+.. highlight:: console
+
+.. author:: Andrej <https://github.com/schoeke>
+.. tag:: lang-node
+.. tag:: project-management
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/wekan.svg
+      :align: center
+
+########
+Wekan
+########
+
+.. tag_list::
+
+Wekan_ - Open Source kanban
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`node <lang-nodejs>`
+  * :manual:`domains <web-domains>`
+  * :lab:`MongoDB <guide_mongodb>`
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+
+License
+=======
+
+Wekan is released under the very permissive `MIT License`_. All relevant information can be found in the GitHub_ repository of the project.
+
+Prerequisites
+=============
+
+We're using :manual:`Nodejs <lang-php>` in the stable version 12:
+
+::
+
+ [isabell@stardust ~]$ $ uberspace tools version use node 12
+ Selected Node.js version 12
+ The new configuration is adapted immediately. Minor updates will be applied automatically.
+ [isabell@stardust ~]$
+
+The domain you want to use should be already set up:
+
+.. include:: includes/web-domain-list.rst
+
+We'll also need :lab:`MongoDB <guide_mongodb>`, so follow the MongoDB guide and come back when it's running.
+
+Let's create a password for a new database:
+
+.. code-block::
+
+ [isabell@stardust ~]$ < /dev/urandom tr -dc _A-Z-a-z-0-9 | head -c${1:-15};echo;
+ randompassword
+ [isabell@stardust ~]$
+
+Once we have that, let's create the new database for Wekan itself.
+
+.. code-block:: 
+
+ [isabell@stardust ~]$ mongo
+ MongoDB shell version v4.4.1
+ connecting to: mongodb://127.0.0.1:21017/admin
+ Implicit session: session { "id" : UUID("6fd371f6-e1fa-461c-be0c-ea3cbe230a01") }
+ MongoDB server version: 4.4.1
+ > use wekan
+ > db.createUser(
+    {
+      user: "wekan",
+      pwd: "randompassword",
+      roles: ["readWrite"]
+    }
+   )
+ > exit
+
+Installation
+============
+
+Download and extract .ZIP archive
+-----------------------------------
+
+Check the Wekan website for the `latest release`_ and copy the download link to the wekan-*.*.zip file. Then use ``wget`` to download it. Replace the URL with the one you just copied.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ wget https://releases.wekan.team/wekan-4.43.zip
```

Comment:
That URL 404s, as wekan removes releases rather quickly from their download page. While every guide says to get the latest release instead of the given URL, many users still use the URL state in the guide. Please add a warning box above this alerting users that they really-really need to get the latest one.